consensus/double_sign.go

```diff
@@ -60,8 +65,7 @@ func (consensus *Consensus) checkDoubleSign(recvMsg *FBFTMessage) bool {
 							return true
 						}
 
-						addr, err := subComm.AddressForBLSKey(offender)
-
+						addr, err := subComm.AddressForBLSKey(*offender)
```

Comment:
it will blow up here too. Just return if offender is nil

Reply:
oops. missed return.